src/app/collective-rewards/components/RewardsMetrics/RewardsMetrics.tsx

```diff
@@ -17,19 +14,27 @@ interface RewardsMetricsProps {
 
 export const RewardsMetrics: FC<RewardsMetricsProps> = ({ title, rbtcRewards, rifRewards }) => {
   const { prices } = usePricesContext()
-  const estimatedRewards = Big(rifRewards.toString())
-    .mul(prices[RIF]?.price ?? 0)
-    .plus(Big(rbtcRewards.toString()).mul(prices[RBTC]?.price ?? 0))
-    .div(WeiPerEther.toString())
-    .toString()
+  const rifPrice = prices[RIF]?.price ?? 0
```

Comment:
Why are we splitting this between RIF and RBTC? I don't think this was ever agreed and/or it's part of this task

Reply:
I reversed it :)